usage-collection/src/generated/main/java/com/cloudera/thunderhead/service/common/usage/UsageProto.java

```diff
@@ -42919,7 +42919,7 @@ private CDPEnvironmentsEnvironmentType(
       YARN(4),
       /**
        * <pre>
-       * YCloud
```

Comment:
do we need this?

Reply:
interesting change. I rebased to the master and dont remember touching this